eslint.config.mjs

```diff
@@ -0,0 +1,81 @@
+import jqueryConfig from "eslint-config-jquery";
+import globals from "globals";
+
+export default [
+	{
+		linterOptions: {
+			reportUnusedDisableDirectives: "error"
+		}
+	},
+
+	{
+		ignores: [
+			"app/dist/**",
+			"external/**",
+			"jquery-ui/**",
+			"template/**",
+			"test/fixtures/**",
+			"tmp/**",
+			"*.min.js"
+		]
+	},
+
+	{
+		languageOptions: {
+			sourceType: "commonjs",
+			ecmaVersion: 2022,
```

Comment:
The ecmaVersion was 2018 before. Is this on purpose?

Reply:
Yeah, we're also upgrading Node.js used to v18 which supports ES2022. And as for the browsers, I don't think the Download Builder needs to work in IE so that'd be fine as well. However, we're still minifying using UglifyJS which is ES5-only so I guess we should guard against that for as long as we use UglifyJS. Added now.

Reply:
Ok, sounds good!